docs/management/connectors/action-types/openai.asciidoc

```diff
@@ -21,6 +21,17 @@ You can create connectors in *{stack-manage-app} > {connectors-ui}*.  For exampl
 image::management/connectors/images/gen-ai-connector.png[OpenAI connector]
 // NOTE: This is an autogenerated screenshot. Do not edit it directly.
 
+[IMPORTANT]
+====
+Elastic provides no official support for connecting to the Azure OpenAI service through a proxy.
+However if you must use a proxy,
+ensure that the proxy supports streaming and is SSE-compatible,
+or the connector will timeout when it attempts to communicate with the Azure OpenAI API.
```

Comment:
streaming is indeed helpful for preventing timeouts, but more importantly, we just fail to parse the response if it is not a streamed response (which is newline-delimited JSON)